_posts/en/newsletters/2022-11-09-newsletter.md

```diff
@@ -0,0 +1,335 @@
+  `-mempoolfullrbf` flag from Bitcoin Core's latest release candidate
+  and not plan to release software with that flag, unless (or until)
+  circumstances change on the network."  He notes:
+
+    - *Opt-in RBF already available:* anyone who wants the benefits of
+      RBF should be able to opt-in to it using the mechanism describe in
+      [BIP125][].  Users would only be served by full RBF if there was
+      some reason they couldn't use opt-in RBF.
+
+    - *Full RBF doesn't fix anything that isn't broken in other ways:*
+      a possible case where some users of a multiparty protocol could
+      deny other users the ability to use opt-in RBF was previously
+      [identified][riard funny games], but Daftuar notes that protocol
+      is vulnerable to other cheap or free attacks which full RBF would
+      not solve.
+
+    - *Full RBF takes away options:* "absent any other examples [of
+      problems fixed by fullrbf], it does not seem to me that fullrbf
+      solves any problems for RBF users, who are already free to choose
+      to subject their transactions to BIP 125's RBF policy.  From this
+      perspective, "enabling fullrbf" is really just taking away user
+      choice to opt a transaction into a non-replacement policy regime."
+
+    - *Offering non-replacement doesn't introduce any issues for full nodes:*
+      indeed, it simplifies the handling of long chains of transactions.
+
+    - *We want other network policies that aren't perfectly incentive compatible:*
+       Daftuar uses the proposal for v3 transaction relay (see
+       [Newsletter #220][news220 v3tx]) as an example:
+
+       > Suppose in a few years someone proposes that we add a
+       > "-disable_v3_transaction_enforcement" flag to our software, to
+       > let users decide to turn off those policy restrictions and
+       > treat v3 transactions the same as v2, for all the same reasons
+       > that could be argued today with fullrbf [...]
+       >
+       > [That] would be subversive to making the lightning use case for
+       > v3 transactions work [...] we should not enable users to
+       > disable this policy, because as long as that policy is just
+       > optional and working for those who want it, it shouldn't harm
+       > anyone that we offer a tighter set of rules for a particular
+       > use case.  Adding a way to bypass those rules is just trying to
+       > break someone else's use case, not trying to add a new one.  We
+       > should not wield "incentive compatibility" as a bludgeon for
+       > breaking things that appear to be working and not causing
+       > others harm.
+       >
+       > I think this is exactly what is happening with fullrbf.
+
+    Daftuar ends his email with three questions for those who still want
+    the `mempoolfullrbf` option to be included in Bitcoin Core:
+
+    1. "Does fullrbf offer any benefits other than breaking zeroconf
```

Comment:
Should this be zero-conf? We use that below. I made PR #922 to add this to the style guide.

Reply:
I'm going to leave this as-is since it's in a quote, it's clear, and I don't think it distracts too much that he uses one spelling and we use a different spelling elsewhere in this newsletter.

Reply:
Ah yes, I didn't notice it was in a quote. I agree.